lte/gateway/python/integ_tests/s1aptests/test_3485_timer_for_dedicated_bearer_with_mme_restart.py

```diff
@@ -0,0 +1,170 @@
+            s1ap_types.ueDetachType_t.UE_SWITCHOFF_DETACH.value,
+        ]
+        wait_for_s1 = [True, False]
+        self._s1ap_wrapper.configUEDevice(num_ues)
+
+        for idx in range(num_ues):
+            req = self._s1ap_wrapper.ue_req
+            print(
+                '********************** Running End to End attach for ',
+                'UE id ',
+                req.ue_id,
+            )
+            # Now actually complete the attach
+            attach = self._s1ap_wrapper._s1_util.attach(
+                req.ue_id,
+                s1ap_types.tfwCmd.UE_END_TO_END_ATTACH_REQUEST,
+                s1ap_types.tfwCmd.UE_ATTACH_ACCEPT_IND,
+                s1ap_types.ueAttachAccept_t,
+            )
+
+            addr = attach.esmInfo.pAddr.addrInfo
+            default_ip = ipaddress.ip_address(bytes(addr[:4]))
+
+            # Wait on EMM Information from MME
+            self._s1ap_wrapper._s1_util.receive_emm_info()
+
+            time.sleep(2)
+            print(
+                '********************** Adding dedicated bearer to IMSI',
+                ''.join([str(idx) for idx in req.imsi]),
+            )
+
+            # Create default flow list
+            flow_list = self._spgw_util.create_default_ipv4_flows()
+            self._spgw_util.create_bearer(
+                'IMSI' + ''.join([str(idx) for idx in req.imsi]),
+                attach.esmInfo.epsBearerId,
+                flow_list,
+            )
+
+            response = self._s1ap_wrapper.s1_util.get_response()
+            self.assertEqual(
+                response.msg_type, s1ap_types.tfwCmd.UE_ACT_DED_BER_REQ.value,
+            )
+            print(
+                '*******************Received first Activate dedicated bearer '
+                'request',
+            )
+
+            print('***** Restarting MME service on gateway')
+            self._s1ap_wrapper.magmad_util.restart_services(['mme'])
+
+            for j in range(20):
+                print('Waiting for', j, 'seconds')
+                time.sleep(1)
+
+            response = self._s1ap_wrapper.s1_util.get_response()
+            act_ded_ber_ctxt_req = response.cast(
+                s1ap_types.UeActDedBearCtxtReq_t,
+            )
+            self._s1ap_wrapper.sendActDedicatedBearerAccept(
+                req.ue_id, act_ded_ber_ctxt_req.bearerId,
+            )
+            print('*******************Send Activate dedicated bearer accept')
+            time.sleep(4)
+
+            print(
+                '********************** Deleting dedicated bearer for IMSI',
+                ''.join([str(idx) for idx in req.imsi]),
+            )
+            self._spgw_util.delete_bearer(
+                'IMSI' + ''.join([str(idx) for idx in req.imsi]), 5, 6,
+            )
+            """During wait time, mme may send multiple times Activate EPS bearer
+               context request, after sending response one message ignore other
```

Comment:
the sentence does not look correct, did you intend to say `after sending response for the first message, ignore the subsequent messages`?

Reply:
Yes